transforms/no-vars.js

```diff
@@ -0,0 +1,79 @@
+/** 
+ * Transform all your `var`s to `let` and `const`.
+ *
+ * The only catch is it may not catch nested bodies that mutate anything beyond
```

Comment:
I'd prefer to make this script safe. I want to run this at FB and will be unable to inspect individual files. This is a one-time script, can we make it slower but correct?